dashboard/config/blocks/GamelabJr/gamelab_stopCollectingData.json

```diff
@@ -0,0 +1,12 @@
+{
+  "category": "Math",
+  "config": {
+    "color": [
+      258,
+      "0.35",
+      "0.62"
```

Comment:
what do these color values mean? RGB? mostly asking because i'm confused by the float values

Reply:
hue, saturation, value